presto-product-tests/conf/docker/common/standard.yml

```diff
@@ -5,6 +5,7 @@ services:
     image: 'teradatalabs/centos6-java8-oracle:${DOCKER_IMAGES_VERSION}'
     volumes:
       - ../..:/docker/volumes/conf
+      - ../../target:/docker/volumes/logs
```

Comment:
does docker create the directory if it doesn't exist? I didn't see a good place to create that logs directory in target, so I left it just target.